tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.Repository/Matter/MatterRepository.cs

```diff
@@ -1913,7 +1957,12 @@ public string GetMatterProvisionExtraProperties(string contentTypeName, Client c
                         jw.WritePropertyName("originalType");
                         jw.WriteValue(field.TypeAsString);
                         jw.WritePropertyName("defaultValue");
-                        jw.WriteValue(field.DefaultValue);
+                        if (client.Name!=null) {
```

Comment:
@venkatesh127 Add null check for matterExtraPropertiesDefaultValues and also count and also check if the key exists in the dictionary